cron/monthly/collective-report.js

```diff
@@ -181,6 +185,20 @@ const processCollective = collective => {
         return collective;
       });
     })
+    .then(async collective => {
+      const transactionsCsvV2 = await getCollectiveTransactionsCsv(collective, { startDate, endDate });
+      if (transactionsCsvV2) {
+        const csvFilenameV2 = `${collective.slug}-${moment(d).format(dateFormat)}-transactions-v2.csv`;
+        options.attachments = [
```

Comment:
Actually, if we're including this a second report, we'll need to do:
```suggestion
        options.attachments.push(
```

Reply:
Thank you for that. When testing locally, we can't see the attachments in MailDev.

Reply:
You can change the view to the Attachments tab:
![Screenshot from 2021-08-31 16-03-39](https://user-images.githubusercontent.com/2119706/131561109-3aa31896-ae1d-4066-a41d-ed260fba9bc6.png)